tests/lambda_endpoints/test_download_query_api.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from sds_data_manager.lambda_code.SDSCode.download_query_api import lambda_handler
+
+BUCKET_NAME = "test-bucket"
+TEST_FILE = "science_block_20221116_163611Z_idle.bin"
+
+
+@pytest.fixture(autouse=True)
+def setup_s3(s3_client):
+    # Populate the s3 client with a bucket and a file
+    s3_client.create_bucket(
+        Bucket=BUCKET_NAME,
+    )
+    result = s3_client.list_buckets()
+    assert len(result["Buckets"]) == 1
+    assert result["Buckets"][0]["Name"] == BUCKET_NAME
+
+    # upload a file
+    local_filepath = f"tests/test-data/{TEST_FILE}"
```

Comment:
Make this change and the test will pass:
local_filepath = os.path.join(os.path.dirname(__file__), '..', 'test-data', TEST_FILE)

Reply:
This test should be passing already? But, I agree that in general we should move towards Pathlib for path joining and our friends using Windows ;) I've made a similar change, but with pathlib instead.